traffic_ops/traffic_ops_golang/monitoring/monitoring.go

```diff
@@ -115,6 +115,7 @@ func Get(w http.ResponseWriter, r *http.Request) {
 }
 
 func GetMonitoringJSON(tx *sql.Tx, cdnName string) (*Monitoring, error) {
+	fmt.Println("SRIJEET I'm HERE")
```

Comment:
We don't need this line, but you can leave it if you want :smile:

Reply:
Lol, no one saw anything 🤫

Reply:
Also, fixed the `TestGetConfig()` test